### ISSUE TITLE
Translation error / a lack of translation

### ISSUE THREAD

**Issue by sDos280** [LAION-AI/Open-Assistant#3159]
Hi, it seems that some languages on the `Your Account` page seem to lack transition for this part:
![image](https://github.com/LAION-AI/Open-Assistant/assets/107134807/2e24f29d-2111-4044-bc0a-02b93036dea0)

As a Hebrew speaker, I would like to fix at least for Hebrew, Where is the place that this part of "code" is responsible for?

p.s. I would really like to contribute to Open-Assistant, but I get a really hard time diving into the code (probably because I never used HTML/JS before) so any direction to the place that is responsible for that will help...

**Comment by sDos280** [comment]
Ok, so I seem to find this place. it seems that the [leaderboard.json of Hebrew](https://github.com/LAION-AI/Open-Assistant/blob/main/website/public/locales/he/leaderboard.json) file is lacking the `level_progress_message` label, may someone verify this?

If that really the case I will happily send a PR that will add that label.

Thanks in advance.

**Comment by olliestanley** [comment]
> Ok, so I seem to find this place. it seems that the [leaderboard.json of Hebrew](https://github.com/LAION-AI/Open-Assistant/blob/main/website/public/locales/he/leaderboard.json) file is lacking the `level_progress_message` label, may someone verify this?
> 
> If that really the case I will happily send a PR that will add that label.
> 
> Thanks in advance.

Yes, this is probably the case. There may be others missing too - in general users submit translation PRs but as new pages/areas of the website are added they can become updated, so PRs updating them are always welcome :)

**Comment by sDos280** [comment]
Ok, thanks for the fast response! I will send a PR soon.

**Comment by sDos280** [comment]
by the way, is there a place to see what are all the missing labels?

**Comment by olliestanley** [comment]
There is a Python script you can use to identify them faster

https://github.com/LAION-AI/Open-Assistant/blob/main/scripts/frontend-development/find-missing-locales.py

**Comment by sDos280** [comment]
Hi @olliestanley, I have pushed my commits and created a [PR](#3160) but I am not sure that I have done good work, is there a way to run this frontend on my machine? also, it seems that for some translations I needed to use gander-related words. For example: in English, we can use "you" to describe a general woman/man/... (like, not specifically a male/female/...) but in Hebrew, I can't really do that, so I just stick to using the male gender. is that ok?
